config-base-next.js

```diff
@@ -20,5 +21,17 @@ module.exports = {
         'unicorn/no-null': 'warn',
       },
     },
+    {
+      files: ['**/test/**/*'],
+      rules: {
+        '@typescript-eslint/consistent-type-assertions': 'off',
```

Comment:
I don't think this should be off here. We're enabling it in the base config and I think we also want it in the next config.